ui/components/multichain/app-header/app-header.js

```diff
@@ -68,6 +65,7 @@ import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
 import { MINUTE } from '../../../../shared/constants/time';
 import { shortenAddress } from '../../../helpers/utils/util';
 import { toChecksumHexAddress } from '../../../../shared/modules/hexstring-utils';
+import { MetaFoxLogo } from './meta-fox-logo';
```

Comment:
We now import `MetaFoxLogo` and `MetafoxLogo`, should we rename one to be more explicit so it's clear what the difference is?

Reply:
++ 

thoughts on MetaFoxLogo → MultichainMetaFoxLogo?

Reply:
make sense, I updated it to `MultichainMetaFoxLogo`